### ISSUE TITLE
Ceph 17.x: "ceph-mgr ... Module not found: 'k8sevents'"

### ISSUE THREAD

**Issue by eskytthe** [NixOS/nixpkgs#241482]
### Describe the bug
In ceph 17.2.5 (17.x) ceph-mgr starts up with these error messages:
```
monA # [   13.409338] ceph-mgr[1051]: 2023-07-04T12:29:59.432+0000 7febfc0d8e00 -1 mgr[py] Module not found: 'k8sevents'
monA # [   13.410877] ceph-mgr[1051]: 2023-07-04T12:29:59.432+0000 7febfc0d8e00 -1 mgr[py] Traceback (most recent call last):
monA # [   13.411442] ceph-mgr[1051]:   File "/nix/store/crlibi37j0849vp6drg8bmsgj72jm58v-ceph-17.2.5-lib/lib/ceph/mgr/k8sevents/__init__.py", line 1, in <module>
monA # [   13.412096] ceph-mgr[1051]:     from .module import Module
monA # [   13.412393] ceph-mgr[1051]:   File "/nix/store/crlibi37j0849vp6drg8bmsgj72jm58v-ceph-17.2.5-lib/lib/ceph/mgr/k8sevents/module.py", line 70, in <module>
monA # [   13.413047] ceph-mgr[1051]:     from kubernetes.client.models.v1_event import V1Event
monA # [   13.413438] ceph-mgr[1051]: ModuleNotFoundError: No module named 'kubernetes.client.models.v1_event'
monA # [   13.415069] ceph-mgr[1051]: 2023-07-04T12:29:59.436+0000 7febfc0d8e00 -1 mgr[py] Class not found in module 'k8sevents'
monA # [   13.415620] ceph-mgr[1051]: 2023-07-04T12:29:59.436+0000 7febfc0d8e00 -1 mgr[py] Error loading module 'k8sevents': (2) No such file or directory
...
monA # [   16.472283] ceph-mgr[1051]: 2023-07-04T12:30:02.495+0000 7febfc0d8e00 -1 log_channel(cluster) log [ERR] : Failed to load ceph-mgr modules: k8sevents
```

### Steps To Reproduce
1. From nixpkgs master:
2. ~/github/nixpkgs]$ nix-build -A nixosTests.ceph-single-node

### Expected behavior
No startup error messages

### Screenshots
-

### Additional context
-

### Notify maintainers
adev@adev.name ak@formalprivacy.com write@ownrisk.dk kristoffer@microdisko.no

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
~/github/nixpkgs]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.36, NixOS, 23.11 (Tapir), 23.11pre500267.645ff62e09d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.1`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by spacekitteh** [comment]
I'm getting this too, and can no longer access the Ceph dashboard.

**Comment by nh2** [comment]
Good news, that seems now fixed in #281858 !
